src/prr.rs

```diff
@@ -11,9 +13,28 @@ use serde_json::{json, Value};
 
 use crate::parser::{LineLocation, ReviewAction};
 use crate::review::{get_all_existing, Review};
+use regex::{Captures, Regex};
+
+// Use lazy static to ensure regex is only compiled once
+lazy_static! {
+    // Regex for short input. Example:
+    //
+    //      danobi/prr-test-repo/6
+    //
+    static ref SHORT: Regex = Regex::new(r"^(?P<org>[\w\-_]+)/(?P<repo>[\w\-_]+)/(?P<pr_num>\d+)").unwrap();
+
+    // Regex for url input. Url looks something like:
+    //
+    //      https://github.com/danobi/prr-test-repo/pull/6
+    //
+    static ref URL: Regex = Regex::new(r".*github\.com/(?P<org>.+)/(?P<repo>.+)/pull/(?P<pr_num>\d+)").unwrap();
+}
 
 const GITHUB_BASE_URL: &str = "https://api.github.com";
 
+/// The name of the local configuration file
+pub const CONFIG_RC_FILE_NAME: &str = ".prrc";
```

Comment:
Nit: name is a bit odd. Technically if it's an rc file it'd be `.prrrc` but that also looks weird. What about `.prr.toml` ? I'd like to at least keep the `toml` extension so it's obvious what file format it is.

Reply:
Changed with `.prr.toml`. I prefer this too